roles/ssh_hardening/tasks/hardening.yml

```diff
@@ -135,3 +135,9 @@
   when:
     - sshd_disable_crypto_policy | bool
     - ('crypto-policies' in ansible_facts.packages)
+
+- name: Enable or disable sshd service
+  ansible.builtin.service:
+    name: "{{ sshd_service_name }}"
+    enabled: "{{ ssh_server_service_enabled }}"
+  become: true
```

Comment:
Why did you add become here?

Reply:
I've copied the behavior from the `handlers/main.yml` but I've just realized that the `become` is already passed as an arg in `tasks/main.yml`, I'll correct this part.

In the mean time, shouldn't we also check that the package is installed with
```
- name: Install openssh
  ansible.builtin.package:
    name: "{{ ssh_pkg_name }}"
    state: present
```
for the same reasons I listed above? The role is almost complete (from a "I want to have a functioning ssh server or client" POV) and it seems quite strange to install and enable the service outside of this role. I'm curious about your thoughts on that

Reply:
In the past, we always assumed that ssh is installed. But since we're now already changing how ssh is started (see https://github.com/dev-sec/ansible-collection-hardening/blob/master/roles/ssh_hardening/tasks/disable-systemd-socket.yml) I don't see any reason not to install it, too.

Do you want to create an additional PR for this?

Reply:
Indeed, I'll create an MR for that if it's ok for you?

Reply:
Yes, please. :)